### PR TITLE
remove duplicate plugins left by #38761 and #40264

### PR DESCRIPTION
**Pull request by missirol**
#### PR description:

#38761 and #40264 led to the renaming of a few plugin types. The old plugin types were kept in place as duplicates at the request of HLT, to ease the renaming of these plugins in the HLT menus. After the HLT-menu updates integrated in #40641, these duplicate plugins can be removed.

Merely technical. No changes expected.

#### PR validation:

Tested a small number of workflows with `runTheMatrix.py`.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR. If this PR will be backported, please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

File: RecoTracker/TransientTrackingRecHit/python/TTRHBuilders_cff.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelPairs_cfi import *
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelTriplets_cfi import *
 #TransientTRH builder with template
-from RecoLocalTracker.SiPixelRecHits.pixelCPEFastESProducer_cfi import *
+from RecoLocalTracker.SiPixelRecHits.pixelCPEFastESProducerPhase1_cfi import *
```
Comment (mmusich) [inline RecoTracker/TransientTrackingRecHit/python/TTRHBuilders_cff.py:10]: to spare me to do it in another PR changing just one line, would you mind adding the phase2 producer here as well?

see https://github.com/cms-sw/cmssw/pull/40215#discussion_r1039795040

**Comment by missirol** [comment]
@cms-sw/reconstruction-l2 @cms-sw/dqm-l2

This PR targets `CMSSW_13_0_0_pre4` (in order to remove these duplicates already in `13_0_X`).

I don't know if testing any extra wfs is useful, so I leave it to experts to set up and start the tests.

FYI: @cms-sw/hlt-l2

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by emanueleusai** [comment]
+1

**Comment by clacaputo** [comment]
+reconstruction

**Comment by perrotta** [comment]
+1
